eZ/Publish/Core/FieldType/Tests/RichText/Converter/Xslt/_fixtures/xhtml5/edit/027-linkedEmbed.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <section xmlns="http://ez.no/namespaces/ezpublish5/xhtml5/edit">
-  <ezembed href="ezcontent://106" id="embed-id-1" data-ezview="embed" class="embed-class" data-ezalign="left">
-    <ezlink href="ezurl://95#fragment1" target="_blank" id="link-id-1" title="Link title" class="link-class"/>
-    <ezconfig>
-      <ezvalue key="size">medium</ezvalue>
-      <ezvalue key="offset">10</ezvalue>
-      <ezvalue key="limit">5</ezvalue>
-    </ezconfig>
-  </ezembed>
-  <ezembed href="ezlocation://601" id="embed-id-2" data-ezview="line" class="embedClass2" data-ezalign="right">
-    <ezlink href="ezlocation://202" class="linkClass2"/>
-  </ezembed>
+  <div data-ezelement="ezembed" href="ezcontent://106" id="embed-id-1" data-ezview="embed" class="embed-class" data-ezalign="left">
+    <a data-ezelement="ezlink" href="ezurl://95#fragment1" target="_blank" id="link-id-1" title="Link title" class="link-class"/>
```

Comment:
Got it, updated in https://github.com/ezsystems/ezpublish-kernel/commit/4eba9311038733d335fa77cb93177a43a5e3b23f.